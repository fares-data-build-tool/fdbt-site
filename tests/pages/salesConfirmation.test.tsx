import * as React from 'react';
import { shallow } from 'enzyme';
import SalesConfirmation from '../../src/pages/salesConfirmation';

describe('pages', () => {
    describe('confirmation', () => {
        it('should render correctly', () => {
            const tree = shallow(
                <SalesConfirmation
                    salesOfferPackages={[
                        {
                            name: 'A sales offer package',
                            description: 'my way of selling tickets',
                            purchaseLocations: ['at stop', 'website'],
                            paymentMethods: ['cash'],
                            ticketFormats: ['paper'],
                        },
                    ]}
                    ticketDating={{}}
                    csrfToken=""
                    pageProps={[]}
                />,
            );
            expect(tree).toMatchSnapshot();
        });
    });
});
