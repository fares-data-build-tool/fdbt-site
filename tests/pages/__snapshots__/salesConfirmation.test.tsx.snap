// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages confirmation should render correctly 1`] = `
<Component
  description="sales Confirmation page of the Fares Data Build Tool"
  errors={Array []}
  title="Sales Confirmation - Fares Data Build Tool"
>
  <CsrfForm
    action="/api/salesConfirmation"
    csrfToken=""
    method="post"
  >
    <h1
      className="govuk-heading-l"
    >
      Check your answers before sending your sales information
    </h1>
    <ConfirmationTable
      confirmationElements={
        Array [
          Object {
            "content": "A Sales Offer Package",
            "href": "selectSalesOfferPackages",
            "name": "Sales Offer Package",
          },
        ]
      }
      header="Sales Information"
    />
    <input
      className="govuk-button"
      id="continue-button"
      type="submit"
      value="Continue"
    />
  </CsrfForm>
</Component>
`;
